mlir/utils/jenkins/Jenkinsfile

```diff
@@ -107,12 +107,64 @@ pipeline {
                      description: 'Run the performance testing stage')
     }
     stages {
+        stage("Set System Property") {
+            steps {
+                script {
+                     System.setProperty("org.jenkinsci.plugins.durabletask.BourneShellScript.HEARTBEAT_CHECK_INTERVAL", "86400");
+                }
+            }
+        }
+        stage("PR tests") {
```

Comment:
I don't think I agree with this. One of the reasons I had the shared library stage above was to combine the logic for PR and nightly fixed tests, since they only differed by a few flags.

Reply:
I've remove the separate PR stage.